Sources/Core/AST/AST.swift

```diff
@@ -17,6 +17,9 @@ public struct AST {
     /// - Invariant: All referred modules have a different name.
     public var modules: [ModuleDecl.ID] = []
 
+    /// The traits in Hylo's standard library that are known by the compiler.
+    var coreTraits: CoreTraits?
```

Comment:
```suggestion
    public var coreTraits: CoreTraits?
```
For consistency with other properties. Or we should make them all internal.